README.md

```diff
@@ -49,34 +49,6 @@ Thus 'master' will support the upcoming major version and the current stable.
 The latest release (git tag, Puppet Forge) should support current and the
 previous stable release.
 
-### Foreman 1.11 compatibility notes
-
-* `foreman::compute::openstack` and `foreman::compute::rackspace` need
-  `package => 'foreman-compute'` on Foreman 1.11 or older, as the package has
-  been split up in newer versions.
-
-### Foreman 1.8/1.9 compatibility notes
-
-On EL or Amazon, set:
-
-    passenger_ruby         => '/usr/bin/ruby193-ruby',
-    passenger_ruby_package => 'ruby193-rubygem-passenger-native',
-    plugin_prefix          => 'ruby193-rubygem-foreman_',
-
-If using `foreman::plugin::ovirt_provision`, puppetdb or tasks, also set the
-`package` parameter as appropriate to:
-
-    ruby193-rubygem-foreman-tasks
-    ruby193-rubygem-ovirt_provision_plugin
-    ruby193-rubygem-puppetdb_foreman
-
-### Foreman 1.7 compatibility notes
-
-* set `apipie_task => 'apipie:cache'` as Foreman 1.7 packages didn't have
-  precompiled API docs
-* `foreman::compute::ec2` needs `package => 'foreman-compute'` on Foreman 1.7,
```

Comment:
Should we delete these `package` parameters or just leave them be?  At a minimum you should probably also remove the class/param docs saying about the Foreman 1.7/11 package names.